content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Change Notes
+description: Istio 1.10.0 release notes.
```

Comment:
Can you please add a publish date? See https://github.com/istio/tools/pull/1509

Reply:
The publish dates will fix:
```


    Release announcement does not show on "release announcements" in News (saw this was already noticed on docs Slack channel)

```

Reply:
Done, thanks